data/puppet_aio/7.yaml

```diff
@@ -0,0 +1,5 @@
+mcollective::service_ensure: stopped
+mcollective::service_enable: false
+mcollective::manage_bin_symlinks: true
+mcollective::plugin_classes:
+  - choria
```

Comment:
these can move up to common.yaml since nothing now has these values as anything else realistically

Reply:
done